radio/src/gui/480x272/lcd.h

```diff
@@ -261,4 +261,29 @@ inline void lcdDrawBitmapPattern(coord_t x, coord_t y, const uint8_t * img, LcdF
   #define SLOW_BLINK_ON_PHASE          (g_blinkTmr10ms & (1<<7))
 #endif
 
+//OW
```

Comment:
maybe you could remove OW from the code? the code review is far easier now!

Reply:
not sure you have seen https://github.com/opentx/opentx/pull/7702#issuecomment-641234721?